### PR TITLE
Add usage examples to README.md

### PR DESCRIPTION
**Pull request by sparkprime**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,76 @@
...
+```
+
+Depend on a package that is in a subtree of a Github repo (this package also
+happens to bring in a transitive dependency):
+
+```sh
+jb install https://github.com/coreos/prometheus-operator/jsonnet/prometheus-operator
+```
+
+Note that if you are copy pasting from the Github website's address bar,
```
Comment (metalmatze) [inline README.md]: Maybe this one can be italic text

**Comment by sparkprime** [comment]
Gonna merge it?

**Comment by metalmatze** [comment]
I can't merge, but @brancz can :relaxed:

**Comment by brancz** [comment]
Oops need to change that.

**Pull request merged**